pkg/agent/util/ipset/ipset.go

```diff
@@ -0,0 +1,78 @@
+// Copyright 2020 Antrea Authors
```

Comment:
Do you like to re-organize other util packages too like net, ethtool?

Reply:
This patch may have been large enough, I can use another PR to re-organize them since they are not really related, what do you think?

Reply:
Sure.